subsys/testsuite/ztest/src/ztest.c

```diff
@@ -1321,15 +1326,57 @@ static int cmd_shuffle(const struct shell *sh, size_t argc, char **argv)
 
 static int cmd_run_suite(const struct shell *sh, size_t argc, char **argv)
 {
+	struct getopt_state *state;
+	int opt;
+	static struct option long_options[] = {{"repeat_iter", required_argument, 0, 'r'},
```

Comment:
`0` could be changed to `NULL` to keep consistency (`flag` is a pointer).

Reply:
Done.